### PR TITLE
fix test config for min-blob-size

### PR DESCRIPTION
**Pull request by tabokie**
Add an uint64 test flag `min-blob-size` for Titan in titandb_bench and titandb_stress tests.

### CODE REVIEW

**Comment by tabokie** [comment]
PTAL @Connor1996

File: tools/db_bench_tool.cc [hunk 1]
```diff
@@ -730,15 +730,19 @@ DEFINE_uint64(blob_db_max_ttl_range, 86400,
 DEFINE_uint64(blob_db_ttl_range_secs, 3600,
               "TTL bucket size to use when creating blob files.");
 
-DEFINE_uint64(blob_db_min_blob_size, 0,
```
Comment (Connor1996) [inline tools/db_bench_tool.cc]: you should not delete this, it is for blobDB

File: tools/db_bench_tool.cc [hunk 2]
```diff
@@ -730,15 +730,19 @@ DEFINE_uint64(blob_db_max_ttl_range, 86400,
...
 DEFINE_uint64(blob_db_bytes_per_sync, 0, "Bytes to sync blob file at.");
 
 DEFINE_uint64(blob_db_file_size, 256 * 1024 * 1024,
               "Target size of each blob file.");
 
+// Titan Options
+DEFINE_bool(use_titan, true, "Open a Titan instance.");
+
+// Shared Options
+DEFINE_uint64(min_blob_size, 0,
```
Comment (Connor1996) [inline tools/db_bench_tool.cc]:
```suggestion
DEFINE_uint64(titan_db_min_blob_size, 0,
```

**Comment by tabokie** [comment]
PTAL @Connor1996

**Pull request merged**